test/browser_tests/conf.js

```diff
@@ -69,12 +69,31 @@ function _isSauceCredentialSet() {
 function _retrieveProtractorParams( params ) { // eslint-disable-line complexity, no-inline-comments, max-len
   var parsedParams = {};
 
-  if ( _paramIsSet( params.specs ) ) {
+  // If one or more suites are specified, use their specs.
+  if ( _paramIsSet( params.suite ) ) {
+    var suitesArray = params.suite.split( ',' );
+    var specsArray = [];
+    for ( var i = 0, suite; suite = suitesArray[i++]; ) {
+      var suiteSpecs = environment.suites[suite];
+      if ( suiteSpecs.constructor === Array ) {
```

Comment:
Probably safer to use `Array.isArray`. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/isArray